modules/rebel/assets/configs/projectilesConfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "laser1": {
```

Comment:
`laser1`
There you are, in the middle of space, fighting enemies and looting planets, when suddenly you see your greatest enemy, the LittleKnight from Deepspace faction. You pull out your blaster and shoot.. `laser1`s? That kills the entire mood. Be a bit more imaginative with your names.